spec/book_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../book'
+
+describe 'Book' do
+  context 'should create an instance of Book class' do
+    book = Book.new('Hobbit', 'Jhon')
+
+    it 'takes two parameters and returns a Book object' do
+      expect(book.class) == Book
+    end
+  end
+end
```

Comment:
- [ ] as required you should test all the methods in this class you forgot some of them so please make sure to write unit tests for them as well in order to match the requirements 
- missing tests for: 
    -  `add_rental`